src/components/application_manager/test/commands/mobile/set_global_properties_test.cc

```diff
@@ -266,17 +295,54 @@ TEST_F(SetGlobalPropertiesRequestTest,
   command->on_event(event_ui);
   command->on_event(event_tts);
 
-  EXPECT_EQ((*response_to_mobile)[am::strings::msg_params][am::strings::success]
-                .asBool(),
-            false);
   EXPECT_EQ(
-      (*response_to_mobile)[am::strings::msg_params][am::strings::result_code]
-          .asInt(),
-      static_cast<int32_t>(mobile_apis::Result::INVALID_DATA));
+      false,
+      (*response_to_mobile)[strings::msg_params][strings::success].asBool());
+  EXPECT_EQ(
+      static_cast<int32_t>(mobile_apis::Result::INVALID_DATA),
+      (*response_to_mobile)[strings::msg_params][strings::result_code].asInt());
+}
+
+TEST_F(SetGlobalPropertiesRequestTest,
+       Run_MsgParametersContainNotValidData_INVALID_DATA) {
+  SmartObject test_so;
+  test_so[0][strings::msg_params][strings::help_prompt][0][strings::text] =
+      kNotValidString;
+
+  test_so[1][strings::msg_params][strings::timeout_prompt][0][strings::text] =
+      kNotValidString;
+
+  test_so[2][strings::msg_params][strings::vr_help][0][strings::text] =
+      kNotValidString;
+
+  test_so[3][strings::msg_params][strings::vr_help][0][strings::text] =
+      kValidString;
+  test_so[3][strings::msg_params][strings::vr_help][0][strings::image]
+         [strings::value] = kNotValidString;
+
+  test_so[4][strings::msg_params][strings::menu_icon][strings::value] =
+      kNotValidString;
+
+  test_so[5][strings::msg_params][strings::vr_help_title] = kNotValidString;
+
+  test_so[6][strings::msg_params][strings::menu_title] = kNotValidString;
+
+  test_so[7][strings::msg_params][strings::keyboard_properties]
+         [strings::limited_character_list][0] = kNotValidString;
+
+  test_so[8][strings::msg_params][strings::keyboard_properties]
+         [strings::auto_complete_text] = kNotValidString;
+
+  test_so[9][strings::msg_params][strings::keyboard_properties]
```

Comment:
@LevchenkoS Well I can, but don't we usually use the "namespace strings = application_manager::strings" abbreviation for strings?